### DISCUSSION TITLE
In volume mode, press the physical volume key vol-display new content

### DISCUSSION THREAD

**Discussion by O3Oalreadytaken** [plateaukao/einkbro#329] [Ideas]
*Use google translate, Please forgive me if my tone is not good.

In volume mode, press the physical volume key
press vol + page content will move to the right (display new content)
press vol - page content will be to the left (showing older content)
I think it should be opposite.

Just like "open fun pageDownWithNoAnimation" and "open fun pageUpWithNoAnimation()"

scrollBy(-shiftOffset(), 0)
scrollX = max(0, scrollX)
and
scrollBy(shiftOffset(), 0)
scrollX = min(computeHorizontalScrollRange() - width, scrollX)

two paragraphs switch places

("switch up/down direction" do not work for the physical volume buttons, only work on the virtual touch buttons on screen.)

**Comment by plateaukao** [comment]
thanks for pointing out this, I'll add an option in settings.

**Comment by plateaukao** [comment]
it's fixed in 523eca19

**Comment by plateaukao** [comment]
@O3Oalreadytaken 
by the way, do you mean in vertical mode?
Because in normal mode, volume key moves page up and down correctly, instead of moving page left or right.

**Reply by O3Oalreadytaken** [reply]
Yes!
Thank you for making the new version so efficiently
